management/dbaas/helpers.go

```diff
@@ -0,0 +1,44 @@
+package dbaas
+
+import (
+	"testing"
+
+	dbaasClient "github.com/percona/pmm/api/managementpb/dbaas/json/client"
+	"github.com/percona/pmm/api/managementpb/dbaas/json/client/kubernetes"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func registerKubernetesCluster(t *testing.T, kubernetesClusterName string) {
+	registerKubernetesClusterResponse, err := dbaasClient.Default.Kubernetes.RegisterKubernetesCluster(
+		&kubernetes.RegisterKubernetesClusterParams{
+			Body: kubernetes.RegisterKubernetesClusterBody{
+				KubernetesClusterName: kubernetesClusterName,
+				KubeAuth:              &kubernetes.RegisterKubernetesClusterParamsBodyKubeAuth{Kubeconfig: "{}"},
+			},
+			Context: pmmapitests.Context,
+		},
+	)
+	require.NoError(t, err)
+	assert.NotNil(t, registerKubernetesClusterResponse)
+}
+
+func unregisterKubernetesCluster(kubernetesClusterName string) {
+	_, _ = dbaasClient.Default.Kubernetes.UnregisterKubernetesCluster(
```

Comment:
I think we should still check `err`, no?

Reply:
I used it in a place where I already call UnregisterKubernetesCluster as part of the test. In case it's failed on list or other part.